src/main/java/org/folio/dataexp/service/export/strategies/AbstractExportStrategy.java

```diff
@@ -151,7 +151,7 @@ protected void createMarc(Set<UUID> externalIds, ExportStrategyStatistic exportS
       var marc = StringUtils.EMPTY;
       try {
         var marcHoldingsItemsFields = additionalFieldsPerId.getOrDefault(marcRecordEntity.getExternalId(), new MarcFields());
-        marc = jsonToMarcConverter.convertJsonRecordToMarcRecord(marcRecordEntity.getContent(), marcHoldingsItemsFields.getHoldingItemsFields());
+        marc = jsonToMarcConverter.convertJsonRecordToMarcRecord(marcRecordEntity.getContent(), marcHoldingsItemsFields.getHoldingItemsFields(), mappingProfile);
```

Comment:
Does it  makes sense to create a DataClass (similar to a DTO) and pass it to a method ?